### PR TITLE
Initial notification functionality is going to be deprecated (documentation removal by the moment)

### PR DESCRIPTION
**Pull request by fgalan**
- [x] deprecation.md
- [x] remove mentions to initial notification in general in documentation
- [x] CNR

### CODE REVIEW

**Comment by fgalan** [comment]
Upon merging of this PR review opened issues related with initial notification and close them by deprecation. As explained at https://fiware-orion.readthedocs.io/en/master/deprecated/index.html:

> Bugs or issues related with deprecated features and not affecting any other feature are not addressed (they are closed in github.com as soon as they are spotted).

File: doc/manuals/user/walkthrough_apiv2.md [hunk 1]
```diff
@@ -709,17 +709,6 @@ Let's examine in detail the different elements included in the payload:
     not only if pressure changes, but if it changes within the range 700-800. This
     is an advanced topic, see the "Subscriptions" section in the
     [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable/).
--   The throttling element is used to specify a minimum
```
Comment (fgalan) [inline doc/manuals/user/walkthrough_apiv2.md:712]: This is not actually related with the initial notification deprecation, but I have taken the opportunity to remove this from the tutorial walk-through, as it is an advanced feature (and even problematic in some cases).

NTC (informative)

**Comment by fgalan** [comment]
@fisuda it would be great if you could do a pull request on this one (branch fgalan-patch-1) to modify the Japanese translation documents, please. That way, we can merge changes in both languages at the same time upon merging of PR #3857.

Thanks!

**Comment by fisuda** [comment]
I have sent PR #3861 to fgalan-patch-1 branch. It would be great if you could review this PR.

**Comment by fgalan** [comment]
> 
> 
> Upon merging of this PR review opened issues related with initial notification and close them by deprecation. As explained at https://fiware-orion.readthedocs.io/en/master/deprecated/index.html:
> 
> > Bugs or issues related with deprecated features and not affecting any other feature are not addressed (they are closed in github.com as soon as they are spotted).

Issues to be closed upon merging this PR:

* https://github.com/telefonicaid/fiware-orion/issues/2994
* https://github.com/telefonicaid/fiware-orion/issues/2437
* https://github.com/telefonicaid/fiware-orion/issues/591

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**